### ISSUE TITLE
FP S1105 `brace-style`:  reporting on closing brace

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarJS#2856]
Rule S1105 is about open brace only, while eslint's `brace-style` is reporting on both open and close braces.

So we are having following issues
```js
if (cond) { return foo(); } // 'Closing curly brace does not appear on the same line as the subsequent block. '
//                        ^
else { return bar(); }
```

Reported in https://community.sonarsource.com/t/brace-style-rule-s1105-flagging-single-line-if-else-pairs/30865
